web/webkit/src/main/scala/net/liftweb/http/provider/HTTPCookie.scala

```diff
@@ -37,19 +41,27 @@ case class HTTPCookie(name: String,
                       maxAge: Box[Int],
                       version: Box[Int],
                       secure_? : Box[Boolean],
-                       httpOnly: Box[Boolean] = Empty) extends java.lang.Cloneable {
+                       httpOnly: Box[Boolean] = Empty,
+                       sameSite : Box[SameSite.Value] = Empty) extends java.lang.Cloneable {
   override def clone(): HTTPCookie = {
     super.clone()
     copy()
   }
 
   /**
    * Returns a new HTTPCookie that preserve existing member values but sets the httpOnly attribute
-   * @param flagHttpOnly - should the cookie be flagged as HTTP Only (only works in Servlet 3.0 containers)
+   * @param flagHttpOnly - should the cookie be flagged as HTTP Only
```

Comment:
This change is because we're now encoding our own cookies and are no longer dependent on the servlet encoding?

Reply:
Yes, exactly, now with our own new handling of the cookies we do not have the limitation of the Servlet version ... it just works for all versions.